.gitignore

```diff
@@ -2,5 +2,6 @@ node_modules
 localStorage/
 npm-debug.log
 .npmrc
+.idea/
```

Comment:
mebs this could go in global .gitignore instead of the repos? what do you think?

Reply:
yeah, good shout

Reply:
@samblacklock how do I actually include this in the global ignore?

Reply:
@RyanKirby89 found some info here 👌 https://blog.martinhujer.cz/dont-put-idea-vscode-directories-to-projects-gitignore/